project-charter.md

```diff
@@ -20,25 +20,25 @@ The mission of the Project is to:
 
 # **2. Governance Board**
 
-1.  The Governance Board (the “GB”) shall be responsible for overall oversight of the Project and coordination of the efforts of any Technical Committees.
+1.  The Governance Board (the “GB”) shall be responsible for overall oversight of the Project and coordination of the efforts of any Technical Committees and Workgroups.
 
 2.  At inception of the Project, the GB voting members consists of those individuals identified on the Project website (https://chaoss.community). Following inception of the Project, the GB will implement procedures and methodologies for the selection of GB voting members from the contributing community.
 
-3.  The GB will encourage transparency (e.g. publish public minutes). Technical Committees meetings will be open to the public, and can be conducted electronically, via teleconference, or in person.
+3.  The GB will encourage transparency (e.g., publish public minutes). Committee and Workgroup meetings will be open to the public, and can be conducted electronically, via teleconference, or in person.
 
-4.  The GB has the authority to create Technical Committees that may focus on code or non-code efforts to advance the Mission (such as standards, specifications, and/or architecture).
+4.  The GB has the authority to create Committees that may focus on code or non-code efforts to advance the Mission (such as standards, specifications, and/or architecture). The Committees of the Project are the “Software Committee,” responsible for technical oversight of inbound and outbound coding projects, and the “Metrics Committee,” responsible for the collection of technology-agnostic metrics and standards to be developed by the Project, and the GB shall provide coordination of the interrelationship between the Software and Metrics Committees.
 
-5.  At the launch of the project, the initial Technical Committees of the project are the “Software Committee,” responsible for technical oversight of inbound and outbound coding projects, and the “Metrics Committee,” responsible for the determination of technology-agnostic metrics and standards to be developed  by the Project, and  the GB shall provide coordination of the interrelationship between the Software and Metrics Committees. At inception of the Project, the voting members of the each Technical Committee consist of those individuals identified on the Project website (http://\[domain name\]). Following the inception of the Project, each Technical Committee is expected to implement procedures and methodologies for the selection of voting members of the Technical Committees.
+5.  Workgroups are estabished and maintained by Project contributors to advance Project work. Workgroups focus on specific metric, methodological, ethical, and technical issues associated with open source project health. 
```

Comment:
Maybe the GB should have a role in accepting working groups? Otherwise, that part of the project could be a bit out of control... I suggest that anyone who is a contributor may propose a working group (or maybe we should ask for at least a number of people to do that, to ensure that they are going to be active), and that it needs to be approved by the GB. And that the GB may decide later on its continuity, in case of no or very little activity/participation/results/compliance with the charter, or whatever... What do you think?

Reply:
Maybe this is something we can discuss at the next meeting. My thought is that GB is tasked with high-level coordination work. I don't think we need GB involvement at all levels of the project, otherwise we start micromanaging and running out of bandwidth for the GB.

Reply:
I agree with micromanaging, but we could have a problem if somebody starts a "CHAOSS wg about how gnomos eat". With this charter, I don't see how that could be stopped (and I don't think we want that kind of wg either ;-)). Maybe we could just state that workgroups may be established as described now, but can be disbanded by the GB if they feel they don't match the goals of CHAOSS. What do you think?

Reply:
+1 to changing terminology to Working Group. I just commented above:

> D&I WG had a discussion about Workgroup vs. Working Group. After looking up the terms, I am still mostly indifferent but I see a trend towards Working Group as an industry standard.
>
> I seem to be the only one who prefers Workgroup and propose to change it to Working Group.

Reply:
Resolving, since it seems we have consensus on "working group".

Reply:
Oooops. Unresolving, since I realize we still need @germonprez feedback on adding working groups to line 41...